### PR TITLE
#BN-11 feat: change password prompt and update password

### PR DESCRIPTION
**Pull request by kunda4**
# PR Description
user should be prompted to update his/her password according to the suggested timing for Administrator
Such that the information about the last time password reset be recorded and get the way of updating his/her password
# Description of tasks that were expected to be completed
List all the tasks and sub-tasks that were assigned to you Make sure you check every completed task. list them in this style
- [x] Create endpoint of reminding user to update password
- [x] notification should be sent with respect with the time
- [x] when the notification prompt is worked on, the timer restarts for the next
time of update
- [x] when the notification prompt is ignored on, the update password prompt
should be turned on and no further operation is allowed.

# Description of tasks that were expected to be completed and that are not yet completed
- [x] none
# How this may be tested?
Please describe the tests that you ran to verify your functionalities. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.
1. run `npm i` to install all required dependencies
2. run `npm run dev`
3. step 3
4. step 4...
# Screenshots (If appropriate/applicable)
Provide screenshots related to your PR in case it's appropriate and applicable
# Please check this Checklist before you submit your PR:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] My code generates no warnings
- [x] There are no vulnerabilities

### CODE REVIEW

**Review by Niyobuzima — DISMISSED** [review]

**Review by nellykawera — DISMISSED** [review]
well done. Approved this PR

File: src/tests/coupon.test.js
Comment (dushimeemma) [inline src/tests/coupon.test.js:1]: Why is this file deleted?

**Review by dushimeemma — APPROVED** [review]
Good job 🥇 🥇
